infrastructure/main.tf

```diff
@@ -109,3 +109,13 @@ resource "azurerm_key_vault_secret" "lau_case_db_user" {
   name         = "case-backend-app-db-user-flexible"
   value        = "lauuser"
 }
+
+resource "azurerm_key_vault_secret" "access_request_key" {
+  key_vault_id = data.azurerm_key_vault.key_vault.id
+  name         = "access-backend-encryption-key"
+  value        = random_password.password.result
+}
+
+resource "random_password" "password" {
+  length           = 32
```

Comment:
Please add a special characters rule - override_special = "()-_"

I've seen in the past keys not being read or set due to special chars such such as *

Also 32 chars looks a bit excessive :)

Reply:
At the moment the key we are using for backend encryption key in lau-idam-backend is 32 chars long and that's why I put 32. Will change the override_rule